victims/victims.go

```diff
@@ -126,6 +129,42 @@ func (v *VictimBase) DeletePod(clientset kube.Interface, podName string) error {
 	return clientset.CoreV1().Pods(v.namespace).Delete(podName, deleteopts)
 }
 
+// Removes a fixed percentage of pods for the victim
+func (v *VictimBase) DeletePodsFixedPercentage(clientset kube.Interface, killPercentage int) error {
```

Comment:
Can you add validation that `killValue` has a valid value (0-100) and a corresponding unit test.

While you're at that, can you add the same validation to `DeleteRandomPodsMaxPercentage` too. Thanks!